cl/scrapers/management/commands/scrape_caselaw.py

```diff
@@ -0,0 +1,339 @@
+        params["page_size"] = page_size
+        if ordering:
+            params["ordering"] = ordering
+        if court:
+            params["court"] = court
+        if jurisdiction:
+            params["jurisdiction"] = jurisdiction
+        if full_case:
+            params["full_case"] = "true" if full_case else "false"
+        if body_format:
+            params["body_format"] = body_format
+        if last_updated and last_updated_filter:
+            params[f"last_updated__{last_updated_filter}"] = last_updated
+        if reporter_id:
+            params["reporter"] = reporter_id
+        if source:
+            params["source"] = source
+
+    # Directory to store data based on type
+    directory = Path(settings.MEDIA_ROOT, "case.law", type)
+
+    if cursor:
+        # Start from specific cursor
+        params["cursor"] = cursor
+
+    while endpoint:
+        data = get_data(endpoint, params=params)
+        # We only need to pass the params on first request
+        params = {}
+
+        if data:
+            results_count = data.get("count", None)
+            old_endpoint = endpoint
+            endpoint = data.get("next", None)
+            results = data.get("results", [])
+
+            for result in results:
+                # We go through each result
+                id = result.get("id")
+                filename = f"{id}.json"
+                if type == "cases":
+                    volume_number = (
+                        result.get("volume").get("volume_number")
+                        if result.get("volume")
+                        else None
+                    )
+                    first_page = result.get("first_page")
+                    reporter_id = (
+                        result.get("reporter").get("id")
+                        if result.get("reporter")
+                        else None
+                    )
+                    reporter_url = (
+                        result.get("reporter").get("url")
+                        if result.get("reporter")
+                        else None
+                    )
+
+                    reporter_slug = get_reporter_slug(reporter_url)
+
+                    if not reporter_slug:
+                        # No slug from url, use reporter id
+                        reporter_slug = reporter_id
+
+                    filename = f"{first_page}.{id}.json"
+                    # Specific directory tree for cases
+                    directory = Path(
+                        settings.MEDIA_ROOT,
+                        "harvard_corpus",
+                        f"law.free.cap.{reporter_slug}.{volume_number}",
+                    )
+
+                file_path = Path(directory, filename)
+
+                if os.path.exists(file_path):
+                    logger.info("Already captured: %s", file_path)
+                    continue
+
+                Path(directory).mkdir(parents=True, exist_ok=True)
+
+                with open(file_path, "w") as outfile:
```

Comment:
I'm not sure we want to overwrite files that are already on the system.  I would almost prefer 

`with open(file_path, "x") as outfile:` to avoid overwriting files.  I know you have a check above but I'm okay throwing an error if something escapes thru.